source/patterns/02-organisms/site-header/site-header.twig

```diff
@@ -0,0 +1,29 @@
+{%- import 'atoms-html' as html -%}
+
+{%- set attributes = attributes|default({})|merge({class: attributes.class|default([])|merge(['site-header']) }) -%}
+
+<header {{- html.attributes(attributes) }}>
+
+    {%- set logo = logo|default({}) -%}
+    {%- with logo -%}
+
+        {%- set href = logo.href|default('') -%}
+        {%- set image = logo.image|default({})|merge({src: asset(logo.image.src|default('images/libero-logo.svg')), alt: logo.image.alt|default('Site logo')}) -%}
+
+      {%- block logo -%}
+          {%- if href is defined and href|length > 0 -%}
```

Comment:
@nlisgo not sure how idiomatic this is, do you know if there's a better way of doing this in Twig?

Reply:
I think the logo should always have a link attached to it so it can return to the home page. I know not everyone knows about this but it has become the normal so think the if statement should be removed in my opinion.

Reply:
I'll take a look at whether this can be condensed.

Reply:
Thanks. Although depending on https://github.com/libero/pattern-library/pull/97#discussion_r285917773 might not need this clause.

Reply:
Not needed now because of https://github.com/libero/pattern-library/pull/97#discussion_r285938667